code/modules/clothing/rogueclothes/storage.dm

```diff
@@ -17,6 +17,7 @@
 	fiber_salvage = TRUE
 	salvage_amount = 1
 	salvage_result = /obj/item/natural/hide
+	var/datum/wound/artery/artery_wound
```

Comment:
Why does this exist? Aren't wounds supposed to be on people, rather than abstract helper objects?